src/blame.cpp

```diff
@@ -1,15 +1,23 @@
-﻿#include "git2cpp/blame.h"
-
-#include <git2/blame.h>
+#include "git2cpp/blame.h"
 
 namespace git
 {
-    git_blame_hunk const* Blame::get_hunk_byline(size_t lineno) const
+    uint32_t Blame::hunk_count() const
+    {
+        return git_blame_get_hunk_count(blame_.get());
+    }
+
+    const git_blame_hunk* Blame::hunk_byindex(uint32_t index) const
+    {
+        return git_blame_get_hunk_byindex(blame_.get(), index);
+    }
+
+    const git_blame_hunk* Blame::hunk_byline(size_t lineno) const
     {
         return git_blame_get_hunk_byline(blame_.get(), lineno);
     }
 
-    void Blame::Destroy::operator()(git_blame* blame) const
+    void Blame::Destroy::operator() (git_blame * blame) const
```

Comment:
This change is redundant.

Reply:
The * alignment is consistent with other function definitions